openlibrary/plugins/openlibrary/code.py

```diff
@@ -21,11 +21,12 @@
     infogami.config.features = []
 
 from infogami.utils.app import metapage
-from infogami.utils import delegate
+from infogami.utils import delegate, dateutil
```

Comment:
this should be openlibrary.utils import dateutil
```suggestion
from openlibrary.utils import delegate, dateutil
```